File: ts/tsism-lib/src/seminar.ts
export enum SeminarType {
  W_SEMINAR,
  P_SEMINAR,
}

export class Seminar {
  readonly id: number
  readonly name: string
  readonly capacity: number
  readonly type: SeminarType

  constructor(id: number, name: string, capacity: number, type: SeminarType) {
    this.id = id
    this.name = name
    this.capacity = capacity
    this.type = type
  }
}

File: python/Seminar.py
from enum import Enum


class SeminarType(Enum):
    SCIENTIFIC = 1
    PRACTICAL = 2


class Seminar:
    name: str
    id: int
    capacity: int
    seminar_type: SeminarType

    def __init__(self, name: str, id: int, capacity: int, seminar_type: SeminarType):
        self.name = name
        self.id = id
        self.capacity = capacity
        self.seminar_type = seminar_type